packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2658,14 +2671,9 @@ class RenderMouseRegion extends RenderProxyBox {
        _onHover = onHover,
        _onExit = onExit,
        _opaque = opaque,
-       _annotationIsActive = false,
-       super(child) {
-    _hoverAnnotation = MouseTrackerAnnotation(
-      onEnter: _handleEnter,
-      onHover: _handleHover,
-      onExit: _handleExit,
-    );
-  }
+       super(child);
+
+  final Key _annotationKey = _UniqueKey();
```

Comment:
How about using `ObjectKey(this)`?

Reply:
i guess it could conflict with someone else theoretically

Reply:
I think unique key is sufficient here, and also faster than ObjectKey. (Neither is available here anyway due to layering)

Reply:
I'm not worried about conflicting, but I think unique key is sufficient here, and is also faster (neither `UniqueKey` nor `ObjectKey` is available here anyway).

I can use `ObjectKey` for the default value of `MouseTrackerAnnotation` though.

Reply:
We could probably move these to foundation where (some of) the other Keys are defined?

Reply:
I moved `UniqueKey` and `ObjectKey` to foundation (while keeping re-exports in widgets) and removed the 2 custom keys.